citesphere/src/main/java/edu/asu/diging/citesphere/core/util/model/impl/CitationHelper.java

```diff
@@ -122,6 +124,22 @@ public void updateCitation(ICitation citation, CitationForm form, IUser user) {
             }
         }
     }
+    
+    /**
+     * Use updateCitationWithCollection(ICitation, String, IUser) method when moving
+     * citation to a collection
+     * 
+     * @param citation     Citation that has to be updated with collection id.
+     * @param collectionId collectionId is the id of collection that the citation is
+     *                     moved to.
+     * @param iUser        iUser who is accessing Zotero.
+     */
+    @Override
+    public void updateCitationWithCollection(ICitation citation, String collectionId, IUser iUser) {
```

Comment:
also, just call the variable `user`. We typically don't prefix variable names with type indicators.